backend/src/common/logging/Logger.ts

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import LogLevel from './LogLevel';
+import { Service } from 'typedi';
+
+@Service()
+export default class Logger {
```

Comment:
Add a test for `Logger`.
Check if `Logger` is logging with given `LogLevel`s.
Especially pay attention to `MUTE` and `default`.